jaseci_core/jaseci/svc/common.py

```diff
@@ -9,11 +9,13 @@
 from jaseci.actions.live_actions import load_action_config
 from jaseci.svc.actions_optimizer.actions_optimizer import ActionsOptimizer
 from .state import ServiceState as Ss
-from .config import META_CONFIG, KUBERNETES_CONFIG
+from .config import META_CONFIG, KUBERNETES_CONFIG, DATABASE as db, POSTGRES_MANIFEST
```

Comment:
Any particular reason you are renaming DATABASE as db here? It is not a big deal but i feel like it doesn't help with readability.